docs/installation.md

```diff
@@ -57,3 +71,5 @@ root user:
 3. Make sure your desktop users are part of the `plugdev` group, by running
    `gpasswd -a <desktop username> plugdev`. If these users were not assigned to the
    group before, they must re-login for the changes to take effect.
+
+TO BE DESCRIBED:  Actually installing solaar in /usr/bin
```

Comment:
Maybe remove just this.

Reply:
Yeah, but it would be nice to have this information available.  I've never tried to do a full install so I was sort of suggesting that it be done soon.  :-)

Reply:
It's just `pip install .`.

Reply:
What are the arguments?  There should be a bunch of files installed, right?  Is there a requirements file to do everything?

Reply:
Aah, I get it now.  `pip install .` in the top-level directory.  That's causing problems for me - I'll investigate further.

Reply:
Okay, thanks!

Reply:
I think I have some good instructions on installing.  I ran into several of the usual issues using pip to install into system directories, but I think I've got them licked.